cfme/common/provider.py

```diff
@@ -310,7 +310,7 @@ def validate_stats(self, ui=False):
                      {'ui': ui},
                      message="do_stats_match_db",
                      num_sec=1000,
-                     delay=60)
+                     delay=120)
```

Comment:
is images the right thing there for openshift? Maybe I chose the wrong wrapanapi function?